video/out/opengl/video.c

```diff
@@ -3391,7 +3393,8 @@ static void check_gl_features(struct gl_video *p)
     bool have_mglsl = gl->glsl_version >= 130; // modern GLSL (1st class arrays etc.)
     bool have_texrg = gl->mpgl_caps & MPGL_CAP_TEX_RG;
     bool have_tex16 = !gl->es || (gl->mpgl_caps & MPGL_CAP_EXT16);
-    bool have_compute = gl->glsl_version >= 430; // easiest way to ensure all
+    bool have_compute = gl->DispatchCompute;
```

Comment:
Maybe it would be better to cache these capabilities in the `gl->caps` field, though. Basically introducing `MPGL_CAP_COMPUTE` which would be defined as both `GL_ARB_shader_image_load_store` and `GL_ARB_compute_shader` and whatever else is necessary (e.g. 2D arrays), since I think we will eventually need 2D arrays anyway. Then we can drop the more confusing/slower `stride` code again.